apidef/api_definitions.go

```diff
@@ -540,6 +540,14 @@ func (s *StringRegexMap) Check(value string) string {
 	return s.matchRegex.FindString(value)
 }
 
+func (s *StringRegexMap) FindStringSubmatch(value string) []string {
+	return s.matchRegex.FindStringSubmatch(value)
```

Comment:
could you also add caching logic for `FindStringSubmatch` in `tyk/regexp`?

Reply:
Please, check https://github.com/TykTechnologies/tyk/pull/1973/commits/990474b4369a223c92c547d2fbc3d4d22fa58eee - Add caching logic for FindStringSubmatch in tyk/regexp

Reply:
👍